tests/data/specs/feature_integration.rflx

```diff
@@ -12,7 +12,7 @@ package Feature_Integration is
          Enumeration : Enumeration_Type::Priority
             then Integer
                if Enumeration = Enumeration_Type::High;
-         Integer : Integer_Type::Byte
+         Integer : Integer_Type::Line_Size
```

Comment:
I don't understand why this change is here. Shouldn't `Byte` just be a ranged integer now?

Reply:
```diff
diff --git a/tests/data/specs/integer_type.rflx b/tests/data/specs/integer_type.rflx
index e89ce3893..59b63fbad 100644
--- a/tests/data/specs/integer_type.rflx
+++ b/tests/data/specs/integer_type.rflx
@@ -3,7 +3,4 @@ package Integer_Type is
    type Page_Num is range 1 .. 2_000 with Size => 16;
    type Line_Size is range 0 .. 255 with Size => 8;
 
-   type Byte is mod 256;
-   type Hash_Index is mod 64;
-
 end Integer_Type;
```

Changing `Byte` to a range integer would lead to a type with the same definition as the existing `Line_Size` type. As these specs are just for testing, I don't see an advantage in keeping `Byte` and `Line_Size` at the same time.